keyboards/redox/rev1/rev1.h

```diff
@@ -1,66 +1,34 @@
-#ifndef REV1_H
-#define REV1_H
+#pragma once
 
 #include "../redox.h"
 
-//void promicro_bootloader_jmp(bool program);
 #include "quantum.h"
 
 
 #ifdef USE_I2C
 #include <stddef.h>
 #ifdef __AVR__
-	#include <avr/io.h>
-	#include <avr/interrupt.h>
+    #include <avr/io.h>
+    #include <avr/interrupt.h>
```

Comment:
Should be double spaced.

```suggestion
  #include <avr/interrupt.h>
```